docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
     image: rabbitmq:3.6-management-alpine
     ports:
       - "5672"
+      - "15672"
```

Comment:
management port may be unavailable for the app - it should check what it uses, because im may be, e.g. firewall problem, or wrong port, or anything else and management tool will tell you everything is fine

Reply:
then I am reverting it back